crates/node/src/shell/testing/node.rs

```diff
@@ -380,6 +380,21 @@ impl MockNode {
             .0
     }
 
+    pub fn next_masp_epoch(&mut self) -> Epoch {
+        let masp_epoch_multiplier =
+            namada::parameters::read_masp_epoch_multiplier_parameter(
+                &self.shell.lock().unwrap().state,
+            )
+            .unwrap();
+        let mut epoch = Epoch::default();
+
+        for _ in 0..masp_epoch_multiplier {
+            epoch = self.next_epoch();
+        }
```

Comment:
ehhh. can't we just use `checked_add`?

Reply:
I believe we need to run the logic of `next_epoch`, don't we?